### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by jonas-zipprick** [allevo/mkdocs-swagger-plugin#1]
Did an install of this plugin as described in README.md an this happens:

```
>mkdocs build --clean
WARNING -  Config value: 'pages'. Warning: The 'pages' configuration option has been deprecated and will be removed in a future release of MkDocs. Use 'nav' instead.
Traceback (most recent call last):
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Jonas\AppData\Local\Programs\Python\Python37\Scripts\mkdocs.exe\__main__.py", line 9, in <module>
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\mkdocs\__main__.py", line 163, in build_command
    ), dirty=not clean)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\mkdocs\commands\build.py", line 240, in build
    config = config['plugins'].run_event('config', config)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\mkdocs\plugins.py", line 94, in run_event
    result = method(item, **kwargs)
  File "c:\users\jonas\appdata\local\programs\python\python37\lib\site-packages\swagger\swagger.py", line 18, in on_config
    with open(js_filepath, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\users\\jonas\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\swagger\\javascript_swagger/swagger_plug
in.js'
```

If i look into the installation directory the missing file is indeed not present.
I am using pip version `19.2.3`.
Any help would be much appreciated.

**Comment by allevo** [comment]
Hi!
thanks for trying to use this plugin.
I think the issue happens because [here](https://github.com/allevo/mkdocs-swagger-plugin/blob/da874465001f45f0036198fca1b0f47a4d61284e/swagger/swagger.py#L13) "/" is used.

Would you link to send a PR in order the use `os.path.join` instead of use "/" directly?

**Comment by jonas-zipprick** [comment]
Hi allevo,
thanks for your reply.

As I already wrote, the file `c:\\users\\jonas\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\swagger\\javascript_swagger/swagger_plugin.js` is not even present after installing. 
I am therefore very sure that the issue is somewhere in the installation process.

Here is what my installation directory looks like after running `pip install git+https://github.com/allevo/mkdocs-swagger-plugin#egg=SwaggerPlugin`:
![image](https://user-images.githubusercontent.com/8794519/65912880-38ab5a00-e3cf-11e9-943d-b0a855299030.png)

**Comment by allevo** [comment]
Ok
It seems that python doesn't package javascript files by default.
I have fixed in #2 . Could you try in that branch?

**Comment by jonas-zipprick** [comment]
The fix seems to work. After running `pip install git+https://github.com/allevo/mkdocs-swagger-plugin@fix/1#egg=SwaggerPlugin` all the required files are installed.
